src/main.cpp

```diff
@@ -14,15 +22,15 @@ static constexpr auto USAGE =
           naval_fate mine (set|remove) <x> <y> [--moored | --drifting]
           naval_fate (-h | --help)
           naval_fate --version
- Options:
+    Options:
           -h --help     Show this screen.
           --version     Show version.
           --speed=<kn>  Speed in knots [default: 10].
           --moored      Moored (anchored) mine.
           --drifting    Drifting mine.
 )";
 
-int main(int argc, const char **argv)
+int main([[maybe_unused]] int argc, [[maybe_unused]] char *argv[])
```

Comment:
I think these variables are definitely being used. Was this change intended for `src/sdl/main.cpp`?

I like the change to `char *argv[]`, but I'm not sure why we lost the `const`.

Reply:
oh, my mistake! will be fixed